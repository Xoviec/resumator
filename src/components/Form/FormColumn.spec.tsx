import { render } from "@testing-library/react";
import { FormColumn } from ".";

describe("FormColumn", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<FormColumn>test-content</FormColumn>);

    expect(asFragment()).toMatchSnapshot();
  });
});
