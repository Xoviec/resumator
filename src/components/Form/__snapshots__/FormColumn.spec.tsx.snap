// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormColumn renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-5sp1us"
  >
    test-content
  </div>
</DocumentFragment>
`;
